<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finishing_DataEntry</class>
 <widget class="QDialog" name="Finishing_DataEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finishing Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Wasted:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QComboBox" name="cboSection">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Gluing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gluing - M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Peeling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stitching</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Dispatch:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Order No:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Produced:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="txtOrderNo"/>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Stop Time:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Section:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Shift:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="txtDispatch"/>
     </item>
     <item row="7" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="2">
      <widget class="QTimeEdit" name="spinStart"/>
     </item>
     <item row="0" column="2">
      <widget class="QDateEdit" name="dtpDate">
       <property name="displayFormat">
        <string>dd MMM yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Start Time:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="txtProduced"/>
     </item>
     <item row="8" column="2">
      <widget class="QTimeEdit" name="spinStop"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="txtWasted"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cboShift">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Day</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Night</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QTextBrowser" name="txtOrderDetails">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>71</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dtpDate</tabstop>
  <tabstop>cboShift</tabstop>
  <tabstop>txtOrderNo</tabstop>
  <tabstop>txtOrderDetails</tabstop>
  <tabstop>txtProduced</tabstop>
  <tabstop>txtWasted</tabstop>
  <tabstop>txtDispatch</tabstop>
  <tabstop>spinStart</tabstop>
  <tabstop>spinStop</tabstop>
  <tabstop>cboSection</tabstop>
  <tabstop>cmdSave</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Finishing_DataEntry</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
